src/components/event/experimental-api/ExperimentalAPIEventStore.ts

```diff
@@ -25,13 +27,35 @@ export class ExperimentalAPIEventStore {
 
 	@observable scrolledIndex: null | Number = null;
 
-	constructor() {
+	events: EventStoreDefaultStateType;
+
+	constructor(events: EventStoreDefaultStateType) {
+		this.events = events;
 		this.getRootIds();
+		this.loadEvent((events as any)?.selectedEventId);
 	}
 
+	@action
+	loadEvent = async (eventId: string) => {
+		if (eventId) {
+			const data = await this.fetchEvent(eventId);
+			const firstChild = JSON.parse(JSON.stringify(data));
```

Comment:
doesnt look like this line doing anything. are you trying to create a copy of data?